lab-adam/gulpfile.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const gulp = require('gulp');
```

Comment:
You don't have any of these dev dependencies on your package.json, so you cannot run gulp if you clone down your repo.